system/Database/Forge.php

```diff
@@ -403,27 +403,40 @@ public function addField($field)
     /**
      * Add Foreign Key
      *
-     * @param string $fieldName
-     * @param string $tableName
-     * @param string $tableField
-     * @param string $onUpdate
-     * @param string $onDelete
+     * @param string|string[] $fieldName
+     * @param string          $tableName
+     * @param string|string[] $tableField
+     * @param string          $onUpdate
+     * @param string          $onDelete
      *
      * @throws DatabaseException
      *
      * @return Forge
      */
-    public function addForeignKey(string $fieldName = '', string $tableName = '', string $tableField = '', string $onUpdate = '', string $onDelete = '')
+    public function addForeignKey($fieldName = '', string $tableName = '', $tableField = '', string $onUpdate = '', string $onDelete = '')
     {
-        if (! isset($this->fields[$fieldName])) {
-            throw new DatabaseException(lang('Database.fieldNotExists', [$fieldName]));
+        $errorNames = [];
+
+        if (is_array($fieldName)) {
+            foreach ($fieldName as $name) {
+                if (! isset($this->fields[$name])) {
+                    $errorNames[] = $name;
+                }
+            }
+        } elseif (! isset($this->fields[$fieldName])) {
+            $errorNames[] = $fieldName;
         }
```

Comment:
You can just simply force this to array then iterate over. No need for if else condition.
```php
$fieldName = (array) $fieldName;

foreach ($fieldName as $name) {
    if (! isset($this->fields[$name])) {
        $errorNames[] = $name;
    }
}
```